Makefile

```diff
@@ -2,7 +2,7 @@ NAME = ircserv
 
 CC = c++
```

Comment:
It is better to replace CC by CXX as CXX is the implicit rule for the cpp compiler